src/commands/publish/mod.rs

```diff
@@ -1,33 +1,26 @@
-mod krate;
 pub mod preview;
 mod route;
+pub mod worker_bundle;
```

Comment:
I feel like `worker_bundle` should be a member of the new `workers` crate you have created, maybe under a `publish` section as you have done with the old `commands::build` logic.

Reply:
i had a similar thought. considering landing this pr and making the additional refactor separately so we can also get kv config landed.